build_debian.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+echo "================================================"
+echo " Attention - Building snarkOS from source code."
+echo " This will request root permissions with sudo."
+echo "================================================"
+
+# Install Debian dependencies
+
+sudo apt-get update
+sudo apt-get install -y \
+	build-essential \
+	curl \
+	gcc \
+	libssl-dev \
+	make \
+	pkg-config \
+        sudo \
+	tmux \
+	xz-utils \
+	ufw
+
+# Install Rust
+
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+source $HOME/.cargo/env
+
+# Install snarkOS
+# cargo clean
+cargo install --locked --path .
+
+echo "=================================================="
+echo " Attention - Please ensure ports 4133 and 3033"
```

Comment:
We should update the ports as well as we are using 4130 and 3030 for all networks. (Should we consider using different ports for different networks?)